src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -171,16 +178,30 @@ public function getTaxTotal(): string
 
     public function hasShippingCharge(string $shippingCharge): bool
     {
-        $shippingChargesText = $this->getElement('shipping_charges')->getText();
+        $shippingChargesText = substr($this->getElement('shipping_adjustment_name')->getText(), 0, -1) . ' ' . $this->getElement('shipping_charges')->getText();
```

Comment:
In more advanced cases (anything above simple two strings concatenation), we should use sprintf
```suggestion
        $shippingChargesText = sprintf(
        	'%s %s',
        	substr($this->getElement('shipping_adjustment_name')->getText(), 0, -1), 
        	$this->getElement('shipping_charges')->getText()
        );
```